autogen/main/versions.tf.tmpl

```diff
@@ -20,22 +20,11 @@
 terraform {
   required_version = ">=0.13"
 
-{% if beta_cluster %}
```

Comment:
Any reason for removing this? Do we use `google-beta` for any resources within the non beta modules?

Reply:
That was a miss. I've added that check back in.